test/verify/check-pages

```diff
@@ -633,12 +633,21 @@ OnCalendar=daily
         b.key_press("stuff/")
         b.wait_in_text("#file-autocomplete-widget li:nth-of-type(5) button", "other")
 
+        # Create test folder with known files
+        m.execute("mkdir /home/admin/newdir")
+        m.execute("mkdir /home/admin/newdir/dir1")
+        m.execute("mkdir /home/admin/newdir/dir2")
+        m.execute("touch /home/admin/newdir/file1")
+        m.execute("touch /home/admin/newdir/file2")
         # test pre-selected autocomplete widget
-        b.wait_val("div#demo-file-ac-preselected input", "/home/admin/.bashrc")
+        b.wait_val("div#demo-file-ac-preselected input", "/home/admin/newdir/file1")
         b.click("div#demo-file-ac-preselected input")
-        paths = ["/home/admin/", "/home/admin/.bash_logout", "/home/admin/.bash_profile", "/home/admin/.bashrc", "/home/admin/.ssh"]
+        b.click("div#demo-file-ac input")
+        b.click("div#demo-file-ac-preselected input")
+        paths = ["/home/admin/newdir", "/home/admin/newdir/dir1", "/home/admin/newdir/dir2", "/home/admin/newdir/file1", "/home/admin/newdir/file2"]
         for i in range(5):
             b.wait_in_text(f"#file-autocomplete-widget-preselected li:nth-of-type({i+1}) button", paths[i])
+        m.execute("rm -r /home/admin/newdir")
```

Comment:
No need to do this. home directory is always cleaned-up. 
https://github.com/cockpit-project/cockpit/blob/main/test/common/testlib.py#L1499